api-reference/beta/resources/onpremisespublishing.md

```diff
@@ -6,12 +6,14 @@
 ### Properties
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
-|customDomainCertificate|String|Details of the certificate associated with the applicaiton when a custom domain is in use. Null when using the default domain.|
 |externalAuthenticationType|String|Details the pre-authentication setting for the application Possible values are: `passthru`, `aadPreAuthentication`.|
 |externalUrl|String|The published external url for the application. For example https://intranet-contoso.msappproxy.net/  |
 |internalUrl|String|The internal url of the application. For example http://intranet/|
 |isOnPremPublishingEnabled|Boolean|Indicates if the application is currently being published or not.|
-|isTranslateHostHeaderEnabled|Boolean|Indicates if the application should translate urls in the reponse headers. This includes setting the correct site for cookies.|
+|isTranslateHostHeaderEnabled|Boolean|Indicates if the application should translate urls in the response headers. This includes setting the correct site for cookies.|
+|verifiedCustomDomainKeyCredential|[keyCerdential](keyCerdential.md)|PFX file containing the certificate to be used with the published application. Used only for POST operations|
+|verifiedCustomDomainKeyCredential|[passwordCredential](passwordCredential.md)|Password associated with the PFX file. Used only in POST operations|
```

Comment:
Copy/paste error.  I think the property name should be verifiedCustomDomainPasswordCredential?